framework/src/meshmodifiers/MeshModifier.C

```diff
@@ -19,6 +19,9 @@ InputParameters validParams<MeshModifier>()
 {
   InputParameters params = validParams<MooseObject>();
   params.addParam<std::vector<std::string> >("depends_on", "The MeshModifiers that this modifier relies upon (i.e. must execute before this one)");
+  params.addParam<bool>("force_prepare", false,
```

Comment:
Is that documented somewhere?  This is fairly complicated... there needs to be at least a little bit of documentation on the Wiki explaining what this thing is...